x-pack/plugins/uptime/server/graphql/pings/schema.gql.ts

```diff
@@ -185,7 +185,7 @@ export const pingsSchema = gql`
     "The timestamp of the ping's creation"
     timestamp: String!
     "Milliseconds from the timestamp to the current time"
-    millisFromNow: Int
+    millisFromNow: String
```

Comment:
Eventually I'd like to return an actual string rather than a number as string. But I'd say it's probably out of scope for the purposes of this PR, and shouldn't be added post-FF anyway, so that looks good.